radio/src/lua/api_general.cpp

```diff
@@ -578,6 +578,72 @@ static int luaGetValue(lua_State * L)
 }
 
 /*luadoc
+@function getUnit(telemetry source)
+
+Return an integer representing the unit used
+
+@Return an unsigned integer representing the unit used or nil if no valid sensor was found
+
+| 2.2 Unit  | Sound        |
+| --- | ---                |
+| 0   | (no unit played)   |
+| 1   | Volts              |
+| 2   | Amps               |
+| 3   | Milliamps          |
+| 4   | Knots              |
+| 5   | Meters per Second  |
+| 6   | Feet per Second    |
+| 7   | Kilometers per Hour|
+| 8   | Miles per Hour     |
+| 9   | Meters             |
+| 10  | Feet               |
+| 11  | Degrees Celsius    |
+| 12  | Degrees Fahrenheit |
+| 13  | Percent            |
+| 14  | Milliamp Hours     |
+| 15  | Watts              |
+| 16  | Milliwatts         |
+| 17  | DB                 |
+| 18  | RPM                |
+| 19  | Gee                |
+| 20  | Degrees            |
+| 21  | Radians            |
+| 22  | Milliliters        |
+| 23  | Fluid Ounces       |
+| 24  | Hours              |
+| 25  | Minutes            |
+| 26  | Seconds            |
+
```

Comment:
Maybe add reference to the luaSetTelemetry function whihc also documents the value types?